lib/ui/viewer/file/file_app_bar.dart

```diff
@@ -124,7 +124,7 @@ class FileAppBarState extends State<FileAppBar> {
     }
     // only show fav option for files owned by the user
     if (isOwnedByUser && !isFileHidden && isFileUploaded) {
-      actions.add(FavoriteWidget(widget.file));
+        actions.add(IconButton(icon: FavoriteWidget(widget.file), onPressed: null));
```

Comment:
I have added the suggested change by @ashilkn to make your life easier :) @Naveenchand06

Reply:
@mmjsmohit I appreciate that you wanted to help. According to @ashilkn, IconButton has default padding That's why while wrapping it with IconButton It is aligned fine. So, it is enough to add padding instead of IconButton.

I've updated the code again

```diff

- actions.add(IconButton(
-           icon: Padding(
-            padding: const EdgeInsets.symmetric(vertical: 6, horizontal: 7.5),
-           child: FavoriteWidget(widget.file),
-          ),
-          onPressed: null));

+ actions.add(
+         Padding(
+          padding: const EdgeInsets.symmetric(vertical: 6, horizontal: 7.5),
+          child: FavoriteWidget(widget.file),
+     ),

```